ui/src/app/base/types.ts

```diff
@@ -1,13 +1,15 @@
+import type { ValueOf } from "@canonical/react-components";
+
 export type TSFixMe = any; // eslint-disable-line @typescript-eslint/no-explicit-any
 
-export enum SortDirection {
-  ASCENDING = "ascending",
-  DESCENDING = "descending",
-  NONE = "none",
-}
+export const SortDirection = {
```

Comment:
Just nitpicking here. Is there any reason why the constant name and its properties wouldn't follow the same naming convention (either all UPPERCASE or PascalCase)?

Seems like UPPERCASE has been already adopted in some places in this repo so might be worth following the same convention for consistency.
https://github.com/huwshimi/maas-ui/blob/ef53877bc99212d81693756fb75b78bf4e726051/ui/src/app/base/constants.ts#L11-L20

Otherwise, If you want to think of`SortDirection` as enum-like, might want to change it all to PascalCase.

> - Use PascalCase for enum names
> - Use PascalCase for enum member
https://basarat.gitbook.io/typescript/styleguide#enumhttps://basarat.gitbook.io/typescript/styleguide#enum

Reply:
There is a definitely a bit of inconsistency but we've mostly been treating enums like types (as they're somewhat interchangeable) so we've been using PascalCase for types and enums, but enum properties have been uppercase, a good example of this is this file:

https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/store/types/enum.ts.

I suspect we've adopted this style mostly because we inherited the syntax from the legacy codebase, e.g. https://github.com/canonical-web-and-design/maas-ui/blob/master/legacy/src/app/enum.js.

We could definitely revisit the syntax, especially as there is already some inconsistency. That styleguide looks pretty handy, I haven't come across that before.